### PR TITLE
fix tof_sino_ucl_proj_data_info 

### PR DESCRIPTION
**Pull request by pkopka**
Result of my debugging `lm_to_projdata` for ROOT input file.

### CODE REVIEW

**Comment by NikEfth** [comment]
Hi @pkopka, 
Thank you for this. 

Would you like to add a small description?

File: src/include/stir/ProjDataInfoCylindricalNoArcCorr.inl [hunk 1]
```diff
@@ -146,7 +146,7 @@ get_bin_for_det_pair(Bin& bin,
   }
   else
   {
-	bin.timing_pos_num() = -timing_pos_num;
```
Comment (NikEfth) [inline src/include/stir/ProjDataInfoCylindricalNoArcCorr.inl:149]: The reason we had this as such was to align it with what is positive or negative time difference.

File: src/include/stir/ProjDataInfo.inl [hunk 2]
```diff
@@ -110,8 +110,8 @@ ProjDataInfo::get_unmashed_tof_bin(const double delta) const
 
   for (int i = min_unmashed_tof_pos_num; i <= max_unmashed_tof_pos_num; ++i)
   {
-    if (delta >= tof_bin_boundaries_ps[i].low_lim &&
-      delta < tof_bin_boundaries_ps[i].high_lim)
+    if (delta >= tof_bin_unmashed_boundaries_ps[i].low_lim &&
```
Comment (NikEfth) [inline src/include/stir/ProjDataInfo.inl:113]: I know that in theory the tof_bin_unmashed_boundaries_ps should be used. 
But this leads to bugs in the listmode reconstruction.

**Review by NikEfth — COMMENTED** [review]
The changes you made on the Proj_data_info break most tests. Would you like to have a chat?

**Comment by KrisThielemans** [comment]
@pkopka, any chance you can do a separate PR for the MPI stuff? It doesn't belong here, and are not controversial 😉 . However, don't use `int&` parameters there. It tells the user that the parameter will be modified, which isn't the case. Even a `const int&` will be slower than an `int` in many architectures.

**Comment by KrisThielemans** [comment]
@pkopka maybe the MPI stuff isn't so important. It's also being fixed in #17. More important is to know what your test case was that led you to make the other changes. If we can reproduce that test case, we can see if it works now or not, including with listmode recon as @NikEfth mentions.

File: src/include/stir/ProjDataInfo.inl (hunk 2 above)
Comment (KrisThielemans) [reply src/include/stir/ProjDataInfo.inl:113]: I've created #32

File: src/include/stir/ProjDataInfoCylindricalNoArcCorr.inl (hunk 1 above)
Comment (KrisThielemans) [reply src/include/stir/ProjDataInfoCylindricalNoArcCorr.inl:149]: agreed that this has to be kept as-is.

**Comment by KrisThielemans** [comment]
I'm closing this as no response. I've created https://github.com/NikEfth/STIR/issues/37 and https://github.com/NikEfth/STIR/issues/32 such that we don't forget what was changed here.

**Pull request closed without merging**